src/auto/file_i_o_stream.rs

```diff
@@ -0,0 +1,68 @@
+// This file was generated by gir (c6b70b0) from gir-files (469db10)
+// DO NOT EDIT
+
+use Cancellable;
+use Error;
+use FileInfo;
+use IOStream;
+use Seekable;
+use ffi;
+use glib::object::IsA;
+use glib::translate::*;
+use glib_ffi;
+use gobject_ffi;
+use std::mem;
+use std::ptr;
+
+glib_wrapper! {
+    pub struct FileIOStream(Object<ffi::GFileIOStream, ffi::GFileIOStreamClass>): IOStream, Seekable;
```

Comment:
Strange, this file not generated now :(

Reply:
I was quite surprised as well.

Reply:
Oh, forgot that it was `file_io_stream.rs` now

Reply:
@GuillaumeGomez, Please do clean regen.

Reply:
I thought I did? I updated `gir` and ran `make gir`. Is there something else to do?

Reply:
`make gir` don't clean `auto` folder, only `make regen_check` do